content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -255,6 +267,20 @@ $ kubectl label namespace default istio.io/dataplane-mode=ambient
 Congratulations! You have successfully added all pods in the default namespace
 to the mesh. Note that you did not have to restart or redeploy anything!
 
+Check once again the ztunnel logs for the proxy has received the network namespace (netns) information about an ambient application pod, and has started proxying for it:
+
+{{< text syntax=bash snip_id=none >}}
```

Comment:
Instead of snip_id=none, you should generate this snip and then call it in test.sh and confirm that the output contains what the doc says it should, e.g., `received netns, starting proy`

Reply:
missed this one. We are on the same page, I will update this soon.